tests/test_plugin_flask_blueprint.py

```diff
@@ -231,23 +231,44 @@ def ping():
 
 
 @pytest.mark.parametrize(
-    ("test_client_and_api", "prefix"),
+    ("test_client_and_api", "prefix", "doc_prefix"),
     [
-        ({"register_blueprint_kwargs": {}}, ""),
-        ({"register_blueprint_kwargs": {"url_prefix": "/prefix"}}, "/prefix"),
+        pytest.param({"register_blueprint_kwargs": {}}, "", "/apidoc", id="default"),
+        pytest.param(
+            {"register_blueprint_kwargs": {}, "api_kwargs": {"path": ""}},
+            "",
+            "",
+            id="root-doc-prefix",
+        ),
+        pytest.param(
+            {"register_blueprint_kwargs": {"url_prefix": "/prefix"}},
+            "/prefix",
+            "/apidoc",
+            id="blueprint-prefix",
+        ),
+        pytest.param(
+            {
+                "register_blueprint_kwargs": {"url_prefix": "/prefix"},
+                "api_kwargs": {"path": ""},
+            },
+            "/prefix",
+            "",
+            id="blueprint-prefix-with-root-doc-prefix",
+        ),
     ],
     indirect=["test_client_and_api"],
 )
-def test_flask_doc_prefix(test_client_and_api, prefix):
+def test_flask_doc_prefix(test_client_and_api, prefix, doc_prefix):
     client, api = test_client_and_api
 
-    resp = client.get(prefix + "/apidoc/openapi.json")
+    resp = client.get(f"{prefix}{doc_prefix}/openapi.json")
+    assert resp.status_code == 200
     assert resp.json == api.spec
 
-    resp = client.get(prefix + "/apidoc/redoc/")
+    resp = client.get(f"{prefix}{doc_prefix}/redoc/")
     assert resp.status_code == 200
 
-    resp = client.get(prefix + "/apidoc/swagger/")
+    resp = client.get(f"{prefix}{doc_prefix}/swagger/")
     assert resp.status_code == 200
 
     assert get_paths(api.spec) == [
```

Comment:
This fails for a couple of the new test cases added when using empty prefix. `api.spec` seems to not be getting populated. Looked at it a bit, but didn't get far enough to determine why. Any ideas?

Reply:
I guess it's due to https://github.com/0b01001001/spectree/blob/0508b707811a85e8294afa4a14fedd6b9b66ba57/spectree/plugins/flask_plugin.py#L18-L22